pkg/webhook/policy.go

```diff
@@ -235,6 +265,27 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 					"request_username", req.AdmissionRequest.UserInfo.Username,
 				).Info("denied admission")
 			}
+			if ref != nil && ref.Name == "gatekeeper-controller-manager" {
+				annotations := map[string]string{
```

Comment:
should some of these be labels so users can use them for selection?

Reply:
Yea I wish 😅   
Seems all we have are annotations: https://github.com/open-policy-agent/gatekeeper/blob/ae3c79c05cbe510eb68fea06ddd8f5c45443ada9/vendor/k8s.io/client-go/tools/record/event.go#L100-L106

Reply:
LOL ah well, we work with what we have.